README.md

```diff
@@ -7,132 +7,59 @@
-
-1. Issue `alr init --bin myproj` (you can use --lib for a library project).
-2. Enter the folder: `cd myproj`
-3. Check that it builds: `alr build`
-4. Run it: `alr run`
-
-## Dependencies and upgrading ##
-Alire keeps track of a project dependencies in the file `./alire/project_name.toml` file of your project. You may check the one just created in the previous example.
-
-This file can be managed through `alr`:
-
-* `alr with project_name` adds a dependency. You can immediately 'with' its GPR project files and packages in your code.
-* `alr with --del project_name` removes a dependency.
-* `alr with --from yourproject.gpr` reads the given GPR file and adds dependencies specified in comments as alr invocations. For example:
-
-    ```Ada
-    with "xstrings"; -- alr with xstrings
-    project My_Project is
-    ```
-
-Alternatively you can edit the file (example in the works) to add dependencies and then issue:
-
-* `alr update`, which will fetch any additional dependencies in your project.
-
-## Finding available projects ##
-
-For quick listing of projects and its description you can use the `list` command:
-
-* `alr list [substring]`
-
-There's also a search command which provides more details:
-
-* `alr search <substring>` will look for `substring` in project names.
-* `alr search --list` will list the whole catalog.
-
-Even more details are obtained with:
-
-* `alr show <project>`
-
-This last command will show generic information. To see the one that specifically applies to your platform:
-
-* `alr show --native <project>`
-
-## Troubleshooting ##
-
-By default `alr` is quite terse and will hide the output of subprocesses, mostly reporting in case of failure. If you hit any problem, increasing verbosity (`-v` or even `-vv`) is usually enough to get an idea of the root of the problem. Additionally, `-d` will show tracebacks of exceptions.
-
-## Running tests ##
-
-`alr` comes with a test suite for self-checks. See the instructions in the [README](https://github.com/alire-project/alire/blob/master/testsuite/README.md) of the `testsuite` folder.
-
-Additionally, you can test in batch the building of crates in your platform with the `alr test` command. (See `alr test --help` output for instructions.)
 
-## Further reading ##
+Alire requires a recent Ada 2012 compiler. In practice, this currently means
+[GNAT Community](https://www.adacore.com/download) or GNAT FSF 7.2 onward. The
+continuous integration checks are run against Debian stable,Ubuntu LTS, Windows
+and MacOS.
 
-More comprehensive documentation is forthcoming, so stay tuned! Meanwhile, you can inspect [index files](https://github.com/alire-project/alire-index) to get an idea of how projects are included into the catalog.
+Note that, for projects that require platform-provided Ada libraries (such as
+Debian's GtkAda), the platform compiler will be used automatically, so even
+having GNAT Community in older platforms some projects may be unavailable due
+to missing dependencies.
```

Comment:
This is no longer true, because right now we don't modify paths (although it may come back shortly with the environment patch). It should be rephrased to something like 

"Note that platform-provided Ada libraries (such as Debian's GtkAda) require the use of the platform Ada compiler. Otherwise these libraries will be unavailable, potentially making dependent crates unavailable too."